templates/overrides/page/page.html.twig

```diff
@@ -96,7 +97,11 @@
   </main>
 
   {% if page.footer %}
-    <footer class="page__footer">
+    {% if "ecfooterblock" in page.footer|keys %}
```

Comment:
I don't like the if, looks bad, I also see that the header blue bar color is different between EU and EC in netlify, so, we can go ahead and create a configuration for the theme here admin/appearance/settings/oe_whitelabel, where we set if it's a EU ot EC website. With that config variable (and we can assume a default) we can decide in situations like the header and the footer if it should be one or the other. See https://github.com/openeuropa/oe_theme/blob/3.x/oe_theme.theme#L44